hangman.rb

```diff
@@ -0,0 +1,72 @@
+
+
+#array of the words to selected for the game
+hangman_words = ["ruby", "python", "coding", "array", "function", "server", "network", "frontend", "backend", "routing",
```

Comment:
use single quotes instead of double quotes.

Reply:
Resolved